src/site/includes/header.html

```diff
@@ -68,7 +68,7 @@
   {% include "src/site/includes/top-nav.html" %}
 {% endif %}
 
-{% include "src/site/blocks/pittsburgh-beta-banner.liquid" %}
+<!-- {% include "src/site/blocks/pittsburgh-beta-banner.liquid" %} -->
```

Comment:
I think they only want to do this in Staging, correct? Or Prod too?

Reply:
Oh good question! Let me ask :)

Reply:
Confirmed with @KevinMHoffmanUSDS and it is indeed only staging! Fix incoming! :)

Reply:
Updated!